Gems/Atom/Feature/Common/Code/Include/Atom/Feature/CubeMapCapture/CubeMapCaptureFeatureProcessorInterface.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/base.h>
+#include <AzCore/Math/Aabb.h>
+#include <Atom/Utils/StableDynamicArray.h>
+#include <Atom/RPI.Public/FeatureProcessor.h>
+#include <Atom/RPI.Reflect/Image/StreamingImageAsset.h>
+#include <Atom/RPI.Reflect/Image/Image.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        class CubeMapCapture;
+
+        using CubeMapCaptureHandle = AZStd::shared_ptr<CubeMapCapture>;
+        using RenderCubeMapCallback = AZStd::function<void(uint8_t* const* cubeMapTextureData, const RHI::Format cubeMapTextureFormat)>;
+        
+        // CubeMapCaptureFeatureProcessorInterface provides an interface to the feature processor for code outside of Atom
+        class CubeMapCaptureFeatureProcessorInterface
+            : public RPI::FeatureProcessor
+        {
+        public:
+            AZ_RTTI(AZ::Render::CubeMapCaptureFeatureProcessorInterface, "{77C6838D-6693-4CF4-9FFC-8110C4551761}");
+
+            virtual CubeMapCaptureHandle AddCubeMapCapture(const AZ::Transform& transform) = 0;
+            virtual void RemoveCubeMapCapture(CubeMapCaptureHandle& cubeMapCapture) = 0;
+            virtual void SetTransform(const CubeMapCaptureHandle& cubeMapCapture, const AZ::Transform& transform) = 0;
+            virtual void SetExposure(const CubeMapCaptureHandle& cubeMapCapture, float exposure) = 0;
```

Comment:
Question mainly for myself, what are the units of exposure used in the engine? Log2 stops with a bias or something else? Are those units documented somewhere?

Reply:
It's just the standard pow(2.0) exposure tonemapping curve.  Jonny is probably the best person to ask about ACES and EV100.